src/AppInitializer.tsx

```diff
@@ -31,217 +35,245 @@ export type SignInProps = {
 
 const COOKIE_CONFIG_KEY = 'org.sagebionetworks.security.cookies.portal.config'
 
-class AppInitializer extends React.Component<Props, AppInitializerState> {
-  constructor(props: Props) {
-    super(props)
-    this.state = {
-      token: undefined,
-      hasCalledGetSession: false,
-      userProfile: undefined,
-      showLoginDialog: false,
-      synapseRedirectUrl: undefined,
+/** On mount, update the document title and meta description using data from the portal config */
+function useSetDocumentMetadataFromConfig() {
```

Comment:
We were doing many different things on `componentDidMount`, so I tried to turn each into it's own hook/instance of `useEffect`